<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateClassClass</class>
 <widget class="QMainWindow" name="TemplateClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TemplateClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="classNameLabel">
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Class name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="classNameLineEdit">
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="InheritanceFromLabel">
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Inheritance from</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="InheritanceFromLineEdit">
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="InheritanceTypeLabel">
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Inheritance type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="InheritanceTypeComboBox">
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>public</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>private</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>protected</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="classTypeLabel">
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Class type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="classTypeLabelComboBox">
        <property name="font">
         <font>
          <family>Yu Gothic UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Class</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Qt Widget</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Qt Main window</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="okToolButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="headerImageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="headerLabel">
          <property name="text">
           <string>Header</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="headerCodeCopyToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="headerCodeTextEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="soucesImageLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sourcesLabel">
          <property name="text">
           <string>Sources</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sourcesCodeCopyToolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="sourcesCodeTextEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="clearToolButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TemplateClass.qrc"/>
 </resources>
 <connections/>
</ui>
